docker/jupyter/Dockerfile

```diff
@@ -28,4 +26,3 @@ ENV LD_LIBRARY_PATH="/usr/local/lib:/usr/lib/x86_64-linux-gnu:/opt/conda/lib/pyt
 ENV PYTHONPATH="/opt/conda/lib/python3.8/site-packages:/home/jovyan/lib"
```

Comment:
Delete this and the above line like we did in the other repos?

Reply:
yup good looks